docs/docsite/rst/community/documentation_contributions.rst

```diff
@@ -206,7 +206,7 @@ Unfortunately, leftover rST-files from previous document-generating can occasion
 Joining the documentation working group
 =======================================
 
-The Documentation Working Group is just getting started, please visit the `community repo <https://github.com/ansible/community>`_ for more information.
+The Documentation Working Group (DaWGs) meets weekly on Tuesdays on the #ansible-docs channel on freenode IRC. For more information, including links to our agenda and a calendar ed, please visit the `working group page in the community repo <https://github.com/ansible/community/wiki/Docs>`_.
```

Comment:
not sure what you meant by 'calendar ed' here

Reply:
Heh, me neither. Maybe "calendar entry"? or "calendar invite"?